waku/v2/README.md

```diff
@@ -134,3 +134,24 @@ You can change this to `wakunode2`, the Waku v2 node like this:
 make docker-image MAKE_TARGET=wakunode2
 docker run --rm -it statusteam/nim-waku:latest --help
 ```
+## Using Websocket and secure Websockets
+
+Websocket support is hidden under a feature flag and must be explicitly enabled in order to get Websockets support. The default port is 8000.
+
+```
+# Run unsecure Websockets (doesn't require a certificate)
+./build/wakunode2 --websocket-support=true 
+```
+
+Running a secure websocket requires an ssl certificate. We can create a self signed websocket. However, it requires the `openssl` utility. It can be achieved with:
+
+```
+mkdir -p ../ssl_dir/
```

Comment:
I'm confused why this is isn't in $PWD?

I'd expect this to be something like:

`mkdir -p certificates` from project root

Reply:
This is largely for KISS to get started / for semi technical people (Desktop etc). Actual sysadminning would probably store certificates in a dir of their own choosing.